### PR TITLE
docs: provide README for NPM purposes

### PR DESCRIPTION
**Pull request by burakukula**
Since we started the internal testing of the v4 packages, we would like to provide small info for users that those packages are part of pre-release to prevent from using them. They can be found in NPM right now. 
We would rather avoid too extended communication, but still provide some info in the NPM.

NPM is picking up `README.md` files. I moved existing docs to `Component.mdx` which will be picked up by the website and added `README.md` which will be picked up by NPM. WDYT?

### CODE REVIEW

File: packages/forma-36-website/gatsby-node.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ exports.onPostBuild = () => {
 
 exports.createPages = ({ graphql, actions }) => {
   const { createPage } = actions;
-
+  console.log('graphql', graphql);
```
Comment (suevalov) [inline packages/forma-36-website/gatsby-node.js:39]: probably should be deleted, right?

Comment (burakukula) [reply packages/forma-36-website/gatsby-node.js:39]: ups

**Review by suevalov — APPROVED** [review]

**Comment by denkristoffer** [comment]
Is the plan to change this back after release or what?

**Review by gui-santos — APPROVED** [review]
Looks good!

I suggest updating the plop-template as well
so when we create a new v4 package this message will be there from the start

**Comment by burakukula** [comment]
> Changes look good 👍 Is the plan to change this back after release or what?

@denkristoffer i guess we would roll back, not really sure how we would approach it. We can make a decision later I guess.

**Pull request merged**